frontend/src/lib/components/infoBadge/infoBadge.svelte

```diff
@@ -4,9 +4,18 @@
   type $$Props = infoBadgeProps;
   export let text: $$Props['text'] = '';
   export let classes: $$Props['classes'] = 'badge';
+  export let disabled: $$Props['disabled'] = false;
 
-  const baseClasses =
-    'badge badge-sm badge-warning px-[0.2rem] relative text-[0.8rem] d-flex justify-center items-center';
+  let baseClasses = 'badge badge-sm';
+
+  if (!disabled) {
+    baseClasses += ' badge-warning';
+  } else {
+    baseClasses += ' badge-ghost';
+  }
+
+  baseClasses +=
+    ' px-[0.25rem] font-bold relative text-[0.8rem] d-flex justify-center items-center';
```

Comment:
Could this be shortened using a ternary operator?

For example:
```
const baseClasses = `badge badge-sm px-[0.25rem] font-bold relative text-[0.8rem] d-flex justify-center items-center ${disabled ? 'badge-ghost' : 'badge-warning'}`;
```

Why are the rest of the base classes defined after the if clause anyway?